tests/build.go

```diff
@@ -1291,6 +1291,29 @@ cOpy Dockerfile .
 		require.Error(t, cmd.Run(), stdout.String(), stderr.String())
 		require.Contains(t, stdout.String(), "Check complete, 2 warnings have been found!")
 	})
+
+	t.Run("check for Dockerfile name when target is set", func(t *testing.T) {
```

Comment:
Don't think that's the right description

Reply:
Ah, good call! Definitely not the correct usage of the term `target` in this context! Updated.